plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -116,12 +116,12 @@ func (g *graphPopulator) updateNode(oldObj, obj interface{}) {
 func (g *graphPopulator) deleteNode(obj interface{}) {
 	node := obj.(*api.Node)
```

Comment:
it's not necessary to check the node name... if we get a delete event for a zero-length node name, it is harmless in this case, and we have much bigger problems.

however, while you're in here, can you correct this to handle DeletedFinalStateUnknown objects:

```go
func (g *graphPopulator) deleteNode(obj interface{}) {
	if tombstone, ok := obj.(cache.DeletedFinalStateUnknown); ok {
		obj = tombstone.Obj
	}
	node, ok := obj.(*api.Node)
	if !ok {
		glog.Infof("unexpected type %T", obj)
		return
	}
	...
}
```

Reply:
Done. Thanks for your quick review.